pip_review/__main__.py

```diff
@@ -61,6 +60,9 @@ def parse_args():
         '--editables', '-e', action='store_true', default=False,
         help='Also include editable packages in PyPI lookup')
     parser.add_argument(
+        '--user', '-u', action='store_true', default=False,
+        help='Only output packages installed in user-site.')
+    parser.add_argument(
```

Comment:
I think options other than `--verbose`, `--raw`, `--interactive` and `--auto` should not be listed explicitly. Instead, they should simply be forwarded to `pip list`. This way, we will be future-proof.